<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>movies_page</class>
 <widget class="QWidget" name="movies_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">movies_page
{
	background-color: rgb(0, 0, 0);
}
QPushButton
{
	color: rgb(237, 212, 0);
	background-color: rgb(0, 0, 0);
	border:2px solid yellow;
	border-radius:10px;
}

QPushButton:hover
{
	color: rgb(238, 238, 236);
	background-color: rgb(237, 212, 0);
}
QPushButton#btn_cancel_search
{
color: rgb(164, 0, 0);
border:2px solid red;
}
QPushButton:hover#btn_cancel_search
{
	color: rgb(238, 238, 236);
	background-color: rgb(164, 0, 0);
}

QPushButton#btn_search
{
	color: rgb(75, 206, 246);
	border:2px solid blue;
}
QPushButton:hover#btn_search
{
	color: rgb(238, 238, 236);
	background-color:rgb(4, 44, 122);
}
QPushButton#btn_perviouspage
{
	color: rgb(78, 154, 6);
	border:2px solid green;
}
QPushButton#btn_nextpage
{
	color: rgb(78, 154, 6);
	border:2px solid green;
}
QPushButton#btn_firstpage
{
	color: rgb(78, 154, 6);
	border:2px solid green;
}
QPushButton#btn_lastpage
{
	color: rgb(78, 154, 6);
	border:2px solid green;
}
QPushButton:hover#btn_perviouspage
{
	color: rgb(238, 238, 236);
	background-color: rgb(78, 154, 6);
}
QPushButton:hover#btn_nextpage
{
	color: rgb(238, 238, 236);
	background-color: rgb(78, 154, 6);
}
QPushButton:hover#btn_firstpage
{
	color: rgb(238, 238, 236);
	background-color: rgb(78, 154, 6);
}
QPushButton:hover#btn_lastpage
{
	color: rgb(238, 238, 236);
	background-color: rgb(78, 154, 6);
}
QPlainTextEdit
{
	background-color:rgb(223, 225, 106);
	color: rgb(0, 0, 0);
	border-radius:6px;
}
QLineEdit
{
	color: rgb(0, 0, 0);
	background-color: rgb(237, 212, 0);
	border-radius:20px;
}
QLineEdit:hover
{
	background-color: rgb(252, 233, 79);
}
QPushButton#btn_profile
{
	color: rgb(75, 206, 246);
	border:2px solid blue;
}
QPushButton:hover#btn_profile
{
	color: rgb(238, 238, 236);
	background-color:rgb(4, 44, 122);
}
QPushButton#btn_control
{
	color: rgb(75, 206, 246);
	border:2px solid blue;
}
QPushButton:hover#btn_control
{
	color: rgb(238, 238, 236);
	background-color:rgb(4, 44, 122);
}
</string>
  </property>
  <widget class="QPushButton" name="btn_search">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>822</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="movieslayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="0" column="2">
     <widget class="QPlainTextEdit" name="show_movie2">
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>450</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>450</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="show_movie1">
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>450</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>450</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPlainTextEdit" name="show_movie3">
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>450</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>450</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>426</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="nextperlayout">
    <item>
     <widget class="QPushButton" name="btn_perviouspage">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Pervious</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_nextpage">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_firstpage">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>First</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_lastpage">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Last</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_counter">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="searchbox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>590</y>
     <width>371</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_reserve">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Reserve</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_delete">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_edit">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_addnew">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Add New</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_cancel_search">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_profile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_control">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Control</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
